lib/pages/chat/chat_event_list.dart

```diff
@@ -41,10 +42,22 @@ class ChatEventList extends StatelessWidget {
         EventTypes.CallInvite,
       }.contains(e.type),
     )) {
-      return Center(
-        child: controller.room?.isDirectChat ?? true 
-          ? const DirectChatEmptyView()
-          : const GroupChatEmptyView(),
+      return Column(
```

Comment:
IMO, we don't need `Column`

```suggestion
return Center(
  child: SingleChildScrollView(
    padding: const EdgeInsets.symmetric(vertical: 16.0),
    controller: controller.scrollController,
    physics: const ClampingScrollPhysics(),
    child: controller.room?.isDirectChat ?? true
      ? const DirectChatEmptyView()
      : const GroupChatEmptyView(),
  ),
);
```

Reply:
I tried it, and wrong

```
The ParentDataWidget Expanded(flex: 1) wants to apply ParentData of type FlexParentData to a RenderObject, which has been set up to accept ParentData of incompatible type BoxParentData.

Usually, this means that the Expanded widget has the wrong ancestor RenderObjectWidget. Typically, Expanded widgets are placed directly inside Flex widgets.
The offending Expanded is currently placed inside a Center widget.
```